test/functorch/test_vmap_registrations.py

```diff
@@ -115,6 +453,15 @@ def test_register_functorch_batched_decomposition(self, registration):
             "then please register it to the FuncTorchBatched key in another file."
         )
 
+    @dispatch_registrations(
+        "CompositeImplicitAutograd", xfail_not_implemented, filter_vmap_implementable
+    )
+    def test_unimplemented_batched_registrations(self, registration):
+        assert registration in FuncTorchBatchedDecompositionRegistrations, (
+            f"Please add a registration in BatchedDecompositions.cpp for "
+            f"the CompositeImplicitAutograd registration {registration}"
```

Comment:
nit: "Please check that there is an OpInfo that covers the operator and add a registration in BatchedDecompositions.cpp". No OpInfo means there is no test coverage that the decomposition is actually OK with vmap.